src/renderer/components/AircraftSection/Properties/index.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Publisher } from 'renderer/utils/InstallerConfiguration';
+import fs from 'fs';
+import path from 'path';
+import { Directories } from 'renderer/utils/Directories';
+import { Toggle } from '@flybywiresim/react-components';
+import { ArrowLeft } from 'react-bootstrap-icons';
```

Comment:
8:10  error  'ArrowLeft' is defined but never used  @typescript-eslint/no-unused-vars